tests/unit/models/asset-size-printer-test.js

```diff
@@ -40,113 +40,110 @@ describe('models/asset-size-printer', function() {
     });
   }
 
-  beforeEach(function() {
-    return mkTmpDirIn(tmpRoot).then(function(tmpdir) {
-      storedTmpDir = tmpdir;
-      assetDir = path.join(storedTmpDir, 'assets');
-      assetChildDir = path.join(assetDir, 'childDir');
+  beforeEach(async function() {
+    storedTmpDir = await mkTmpDirIn(tmpRoot);
+    assetDir = path.join(storedTmpDir, 'assets');
+    assetChildDir = path.join(assetDir, 'childDir');
 
-      fs.mkdirsSync(assetDir);
-      fs.mkdirsSync(assetChildDir);
+    fs.mkdirsSync(assetDir);
+    fs.mkdirsSync(assetChildDir);
 
-      writeFiles();
-    });
+    writeFiles();
   });
 
   afterEach(function() {
     return Promise.all([remove(storedTmpDir)]);
```

Comment:
the `Promise.all()` here seems unnecessary. you can replace that while you're on it:

```js
afterEach(async function() {
  await fs.remove(storedTmpDir);
});
```